code/game/objects/effects/aliens.dm

```diff
@@ -48,11 +48,11 @@
 
 /obj/effect/xenomorph/spray/Initialize(mapload, duration = 10 SECONDS) //Self-deletes
 	. = ..()
-	START_PROCESSING(SSobj, src)
+	START_PROCESSING(SSfastprocess, src)
```

Comment:
```suggestion
	START_PROCESSING(SSprocessing, src)
```
This one checks every second.

Reply:
Also this shouldn't be calling `Crossed()` in `process()`, but that's a different thing.
Both `Crossed()) and `process()` should be calling something like `acid_burn_feet()`